mods/cnc/maps/scb30ea/scb30ea.lua

```diff
@@ -0,0 +1,376 @@
+CiviliansHelpTeam = { "mtnk", "mtnk", "msam" }
+BaseDefenseTeam = { Def1, Def2, Def3, Def4 }
+PatrolTeam = { Actor103, Actor122, Actor123 }
+PatrolPath = { waypoint2.Location, waypoint3.Location, waypoint4.Location, waypoint5.Location }
+
+CapturableStructures = { "afld", "hand", "hq", "nuke", "silo", "proc", "sam" }
+IonCannonTargets = { { "obli", "gun", "gtwr" }, { "harv" } }
+
+WelcomeTeamCellTrigger = { CPos.New(17,59), CPos.New(18,59), CPos.New(19,59), CPos.New(17,60), CPos.New(18,60), CPos.New(19,60), CPos.New(25,53), CPos.New(25,54), CPos.New(25,55), CPos.New(26,53), CPos.New(26,54), CPos.New(26,55) }
+CivsGDIHelpCellTrigger = { CPos.New(13,7) }
+GDIBaseEntranceCells = { CPos.New(16,5), CPos.New(16,6), CPos.New(16,7), CPos.New(16,8), CPos.New(17,4), CPos.New(17,5), CPos.New(17,6), CPos.New(17,7), CPos.New(17,8), CPos.New(18,4), CPos.New(18,5), CPos.New(19,4), CPos.New(8,15), CPos.New(9,15), CPos.New(10,15), CPos.New(11,15), CPos.New(12,15), CPos.New(13,15), CPos.New(14,15), CPos.New(9,16), CPos.New(10,16), CPos.New(11,16), CPos.New(12,16), CPos.New(13,16), CPos.New(14,16) }
+InnerGDIBaseEntranceCells = { CPos.New(7,5), CPos.New(7,6), CPos.New(7,7), CPos.New(8,6), CPos.New(8,7), CPos.New(9,6), CPos.New(9,7) }
+
+
+CaptureStructures = function(actor)
+	for i = 1, #CapturableStructures do
+		local structures = Nod.GetActorsByType(CapturableStructures[i])
+		if #structures > 0 and not actor.IsDead and not structures[1].IsDead then
+			actor.Capture(structures[1])
+			return
+		end
+	end
+end
+
+
+SendGDIAirstrike = function()
+	if not GDIhq.IsDead and GDIhq.Owner == GDI then
+		local target = GetAirstrikeTarget(Nod)
+		if target then
+			GDIhq.TargetAirstrike(target, Angle.SouthEast)
+		else
+			Trigger.AfterDelay(DateTime.Seconds(5), SendGDIAirstrike)
+		end
+	end
+end
+
+
+OilDerricksAstkSent = false
+OilDerricksAirstrike = function()
+	if not OilDerricksAstkSent then
+		SendGDIAirstrike()
+		OilDerricksAstkSent = true
+	end
+end
+
+
+CivsRunning = false
+RunForHelp = function()
+	if not CivsRunning then
+		Utils.Do(RunForHelpCivs, function(actor)
+			actor.Move(CPos.New(53,45))
+			actor.Move(waypoint6.Location)
+			actor.Move(waypoint4.Location)
+			actor.Move(waypoint9.Location)
+			actor.Move(waypoint2.Location)
+			actor.Move(waypoint12.Location)
+			actor.Move(waypoint6.Location)
+		end)
+		Trigger.OnEnteredFootprint(CivsGDIHelpCellTrigger, function(a, id)
+			if a == RunForHelpCivs[1] or a == RunForHelpCivs[2] then
+				Reinforcements.Reinforce(GDI, CiviliansHelpTeam, { CPos.New(2,9), CPos.New(3,9) }, 30, function(a)
+					a.AttackMove(waypoint2.Location)
+					a.AttackMove(waypoint9.Location)
+					a.AttackMove(waypoint8.Location)
+					IdleHunt(a)
+				end)
+				Trigger.RemoveFootprintTrigger(id)
+			end
+		end)
+		CivsRunning = true
+		local cam = Actor.Create("camera", true, { Owner = Nod, Location = CPos.New(53,44) })
+		Trigger.AfterDelay(125, function() cam.Destroy() end)
```

Comment:
There are some Trigger lines that could be shortened, where a single function is called without arguments. This already seems to be done elsewhere, and more consistency can't hurt.
```suggestion
		Trigger.AfterDelay(125, cam.Destroy)
```